server/app/models/ProductionModel.ts

```diff
@@ -0,0 +1,31 @@
+import { Document, Schema } from 'mongoose';
+
+export interface IProduction extends Document {
+  status: string;
+  orderID: number;
+  component: string;
+  option1: string;
+  option2: string;
+  quantity: string;
+  expectStartDate: Date;
+  expectedEndDate: Date;
+  actualStartDate: Date;
+  actualEndDate: Date;
+  manufacture: string;
+  note: string;
+}
```

Comment:
Aren't a lot of these unneeded? @Mackbellemore 

Like manufacture? Aren't we the manufacturer? What's option1, option2 for? Component?

Reply:
The manufacturer would be which machine is producing the bike. For options, it's an additional feature that they would add such as a different paint color, or any option that ain't specified.